configuration/price/strategy.json

```diff
@@ -0,0 +1,4 @@
+{
```

Comment:
Shouldn't this be network based as well, like the announcements?

Reply:
i wwas thinking that it shouldn't actually. but maybe it doesn't hurt to include that 🤷

Reply:
My thinking was, maybe only Mainnet endpoint has issues and GC is fine, so we wouldn't need to change the setting for both.